README.md

```diff
@@ -1,65 +1,69 @@
-  * Example list of tasks for the Star Wars films example:
-    * Setup `create-react-app` scaffold
-    * Call the Star Wars API and print results
-    * Store API results in React component state
-    * Render React state
-* A plan for the next 3 weeks and what you plan to accomplish each week
-
-## Recommendations
-
-* Keep the scope of your project small. A small, completed project is much better than a large, incomplete project. You can create a list of "nice to have" features that you can work on if you have time (similar to how our assignments have a main list of requirements and "bonus" items).
-* Ensure you have plenty of buffer time in your schedule. Life events pop up, and tasks can take longer than initially estimated.
-
-## Submitting your Project Idea
-
-1. Fork this repository
-2. Add your project name, elevator pitch, wireframes, dependencies, task list, and plan to the Github repo (you can add a markdown file + images or links to images)
-3. Create a pull request
+# Bus Arrival Dashboard
+
+Google, OneBusAway, and other apps offer real-time bus arrival information. The OneBusAway app even displays animation with bus locations.
+
+Sometimes you need something much simpler. I live close to two bus stops and different buses that can get me to my destination (work) stop at each of them. A dashboard listing only the buses I am interested in at only the stops I am interested in would be a very good thing in the morning.
+
+The Bus Arrival Dashboard should group the buses by stop and sort them by arrival time. Color coding the arrival times would help me decide which stop to walk toward.
+
+## Wireframes
+
+### Sign in
+![sign in](wireframes/images/images.001.png)
+
+### Main page
+![main page](./wireframes/images/images.007.png)
+
+### Admin
+![enter bus number](./wireframes/images/images.002.png)
+
+![select direction of travel](./wireframes/images/images.003.png)
+
+![select stop](./wireframes/images/images.004.png)
+
+![add to group](./wireframes/images/images.005.png)
+
+![new route added](./wireframes/images/images.006.png)
+
+## Dependencies
+
+* node-sass
+* OneBusAway API
+* A back end to send requests to OneBusAway API (Rails API app)
+* A host
+* Firebase for authentication and user data
+
+## Tasks
+
+- [ ] Rails backend app
+  - Routes
+    - [ ] Fetch route info for bus
+    - [ ] Fetch arrival info for bus at stop
+- [ ] Find host for Rails app
+  - [ ] Deploy to host
+- [ ] Create React App
+  - [ ] Sign in
+  - [ ] Admin
+  - [ ] Main page
+
+## Plan
+
+### May 23-24 2020
+
+- [x] Create Rails backend API
+
+### May 30-31 2020
+
+- [ ] Find host and deploy Rails API
+- [ ] Create React front end
+  - [ ] Authorization using Firebase
+  - [ ] Data storage using Firebase
+  - [ ] Admin page to add and remove buses and groups
+  - [ ] Main display
+
+### June 6-7 2020
+
+- [ ] Add navigation
+- [ ] Add CSS styles
+- [ ] Add any additional tests
+- [ ] Deploy to host
```

Comment:
Once the front-end, I would double-check to make sure you don't run into any CORS issues when calling the Rails API from the React app. Let Tim or I know if you have any questions about this.